### PR TITLE
Adv/fix graph from edges

### PR DESCRIPTION
**Pull request by antonydellavecchia**
Corrects docs to include graphs from edges, and modifies doc string to properly describe the expected functionality.

### CODE REVIEW

File: src/Combinatorics/Graphs/functions.jl [hunk 1]
```diff
@@ -1097,37 +1097,36 @@ function graph_from_edges(::Type{T},
...
-```
-# Examples 2
-```jldoctest
-julia> G = graph_from_edges(Undirected,[[1,3]])
-Undirected graph with 3 nodes and the following edges:
-(3, 1))
-
+julia> G = graph_from_edges(Directed, [[1,3]], 4)
+Directed graph with 4 nodes and the following edges:
+(3, 1)
```
Comment (antonydellavecchia) [inline src/Combinatorics/Graphs/functions.jl]:
```suggestion
(1,1)
```

File: src/Combinatorics/Graphs/functions.jl [hunk 2]
```diff
@@ -1097,37 +1097,36 @@ function graph_from_edges(::Type{T},
...
-```
-# Examples 2
-```jldoctest
-julia> G = graph_from_edges(Undirected,[[1,3]])
-Undirected graph with 3 nodes and the following edges:
-(3, 1))
-
+julia> G = graph_from_edges(Directed, [[1,3]], 4)
+Directed graph with 4 nodes and the following edges:
+(1,1)
```
Comment (benlorenz) [inline src/Combinatorics/Graphs/functions.jl]:
```suggestion
(1, 3)
```

File: src/Combinatorics/Graphs/functions.jl [hunk 3]
```diff
@@ -1097,37 +1097,36 @@ function graph_from_edges(::Type{T},
...
+                          n_vertices::Int=-1) where {T <: Union{Directed, Undirected}}
+  return graph_from_edges(T, collect(edges), n_vertices)
+end
 
 @doc raw"""
-  graph_from_edges(::Type{T}, edges::Vector{Vector{Int}}) where {T <:Union{Directed, Undirected}}
-
-Creates a graph from a vector of edges. Optionally, you could input the number of vertices, but if this number is lower than the maximum vertex in the edges, this argument will be ignored.
+     graph_from_edges(edges::Vector{Vector{Int}})
+     graph_from_edges(::Type{T}, edges::Vector{Vector{Int}}, n_vertices::Int=-1) where {T <:Union{Directed, Undirected}}
```
Comment (benlorenz) [inline src/Combinatorics/Graphs/functions.jl]:
```suggestion
    graph_from_edges(edges::Vector{Vector{Int}})
    graph_from_edges(::Type{T}, edges::Vector{Vector{Int}}, n_vertices::Int=-1) where {T <:Union{Directed, Undirected}}
```

File: src/Combinatorics/Graphs/functions.jl [hunk 4]
```diff
@@ -1097,37 +1097,36 @@ function graph_from_edges(::Type{T},
...
+  return graph_from_edges(T, collect(edges), n_vertices)
+end
 
 @doc raw"""
-  graph_from_edges(::Type{T}, edges::Vector{Vector{Int}}) where {T <:Union{Directed, Undirected}}
-
-Creates a graph from a vector of edges. Optionally, you could input the number of vertices, but if this number is lower than the maximum vertex in the edges, this argument will be ignored.
+    graph_from_edges(edges::Vector{Vector{Int}})
+    graph_from_edges(::Type{T}, edges::Vector{Vector{Int}}, n_vertices::Int=-1) where {T <:Union{Directed, Undirected}}
+Creates a graph from a vector of edges. There is an optional input for number of vertices, `graph_from_edges`  will
```
Comment (lgoettgens) [inline src/Combinatorics/Graphs/functions.jl]:
```suggestion
    graph_from_edges(edges::Vector{Vector{Int}})
    graph_from_edges(::Type{T}, edges::Vector{Vector{Int}}, n_vertices::Int=-1) where {T <:Union{Directed, Undirected}}

Creates a graph from a vector of edges. There is an optional input for number of vertices, `graph_from_edges`  will
```
empty line between signature and text

**Review by lkastner — APPROVED** [review]

**Pull request merged**